[BUG]: missing ncurses

While building the resen-core image, specifically while building the cdf library:

    Step 14/43 : RUN /bin/bash -c 'bash install_CDF.sh &&                   echo "if [ -z \$CDF_LIB ]; then source /usr/local/bin/definitions.B; fi" >>  /home/jovyan/.bashrc'

I noticed the following error:

    In file included from ../include/widgets.h:36:0,
                     from ../include/cdfxp.h:36,
                     from cdfxp.c:44:
    ../include/windoz.h:81:16: fatal error: ncurses.h: No such file or directory
     #      include <ncurses.h>
                    ^~~~~~~~~~~
    compilation terminated.
    Makefile:170: recipe for target 'cdfxp.o' failed
    make[1]: *** [cdfxp.o] Error 1
    Makefile:803: recipe for target 'all.build' failed
    make: *** [all.build] Error 2

I think this is resolved by installing `libncurses5-dev`.

--- a/resen-core/Dockerfile
+++ b/resen-core/Dockerfile
@@ -34,7 +34,8 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     mlocate \
     less \
-    vim
+    vim \
+    libncurses5-dev
 
 # custom scripts
 COPY resources/start-singleuser-ingeo.sh /usr/local/bin

